test/integration/setup_test.go

```diff
@@ -76,282 +49,24 @@ func TestMain(m *testing.M) {
 		os.Exit(exitCode)
 	}()
 
-	clientset, err := k8sutils.MustGetClientset()
+	clientset, err := kubernetes.MustGetClientset()
 	if err != nil {
 		return
 	}
 
 	ctx := context.Background()
-	if installopt := os.Getenv(envInstallCNS); installopt != "" {
+	if installopt := os.Getenv(kubernetes.EnvInstallCNS); installopt != "" {
```

Comment:
Can we match this to the same pattern as the load*() functions?

Reply:
Done